general/clients/installing-kodi.md

```diff
@@ -1,61 +1,101 @@
 ---
 uid: clients-kodi
-title: Kodi
+title: Installing Kodi Add-On
 ---
 
-# Kodi
+# Installing Kodi Add-On
+
+> [!NOTE]
+> It's highly recommended to install the `Kodi Sync Queue` plugin into the Jellyfin server as well.
+> This will keep your media libraries up to date without waiting for a periodic re-sync from Kodi.
+
 
 ## Installation Process
 
-1. Download the repository installer [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip).
+### Install Add-On Repository
+
+The recommended install method is to use the official Jellyfin add-on repository.  This allows for easy install of the Jellyfin for Kodi add-on, as well as automatically keeping the add-on up to date with the latest version.  Any other Jellyfin related add-ons that may be built in the future will also be available in this repository.
+
+#### General Use Devices (PCs and Tablets)
+
+1. Download the repository installer found [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip).
     * It will be saved as `repository.jellyfin.kodi.zip`
 2. Install the Jellyfin repository.
-    * Open Kodi and navigate to "Add-On Browser"
+    * Open Kodi, go to the settings menu,  and navigate to "Add-On Browser"
     * Select "Install from Zip File"
-        * If prompted, enter settings and enable "Unknown Sources", then go back
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
     * Select the newly downloaded file and it will be installed
-3. Install Jellyfin for Kodi.
+
+#### "Embedded" Devices (Android TV, Firestick, and other TV Boxes)
+
+1. Open Kodi, go to the settings menu, and navigate to "File manager"
+    * Select "Add source"
+    * In the text box, enter `https://repo.jellyfin.org/releases/client/kodi/`
+    * Enter a name for the data source, such as "Jellyfin Repo" and select Ok
+2. From the settings menu, navigate to "Add-On Browser"
+    * Select "Install from Zip File"
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
+    * Select the data source you just added
+    * Install `repository.jellyfin.kodi.zip`
+
+### Install Jellyfin for Kodi Add-On
```

Comment:
Can we bump this section up to LOC28? Trying to install via this and I was confused by having to skip the Embedded devices section. 

Better yet, maybe add a step to `See Install Jellyfin for Kodi Add-On`

Reply:
That's why I used different sized headings, though that could be made clearer with a line "there's two options to install the repository".  I'd like to avoid a situation where you have to jump up and down the page like in the current doc to find the info you need

Reply:
i'm fairly unfamiliar with kodi so i'm fairly dependent on the docs. once I had installed the zip file, i went back to the docs trying to find the next section and `Install Jellyfin for Kodi Add-On` didnt really seem like it's the next section to perform so i became quite confused as to what to do next lol.

Reply:
I added more links on next steps